src/Mobs/Path.cpp

```diff
@@ -67,22 +64,8 @@ cPath::cPath(
 
 	m_NearestPointToTarget = GetCell(m_Source);
 	m_Status = ePathFinderStatus::CALCULATING;
-	m_StepsLeft = a_MaxSteps;
-
-	ProcessCell(GetCell(a_StartingPoint), nullptr, 0);
-	m_Chunk = nullptr;
-}
-
 
-
-
-
-cPath::~cPath()
-{
-	if (m_Status == ePathFinderStatus::CALCULATING)
-	{
-		FinishCalculation();
-	}
+	ProcessCell(GetCell(m_Source), nullptr, 0);
```

Comment:
Here is the bug:

`ProcessCell(GetCell(a_StartingPoint)`
instead of
`ProcessCell(GetCell(m_Source)`

Where m_Source is a floored and adjusted a_StartingPoint.